tests/foreman/api/test_role.py

```diff
@@ -100,7 +100,48 @@ def test_positive_update(self):
 class CannedRoleTestCases(APITestCase):
     """Implements Canned Roles tests from API"""
 
-    @stubbed()
+    def create_org_admin(self, name=None, orgs=None, locs=None):
+        """Helper function to create org admin role for particular
+        organizations and locations by cloning 'Organization admin' role.
+
+        :param str name: The name of cloned Org Admin role
+        :param list orgs: The list of organizations for which the org admin is
+            being created
+        :param locs: The list of locations for which the org admin is
+            being created
+        :return dict: This function returns dict representation of cloned role
+            data returned from 'clone' function
+        """
+        name = gen_string('alpha') if not name else name
+        orgs = [] if not orgs else orgs
+        locs = [] if not locs else locs
+        default_org_admin = entities.Role().search(
+            query={'search': u'name="Organization admin"'})
+        org_admin = entities.Role(id=default_org_admin[0].id).clone(
+            data={
+                'role': {
+                    'name': name,
+                    'organization_ids': orgs,
+                    'location_ids': locs
+                }
+            }
+        )
+        self.assertEqual(org_admin['name'], name)
+        return org_admin
```

Comment:
result which user is receiving after creating an entity is usually entity read from the server (by `EntityName(id=#).read()` command), not some helper's raw output. Please read created entity and return it instead to avoid confusion and different styles like `self.assertNotEqual(org_admin['id'], role.id)`